SwiftCheckTests/NSDataSpec.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NSDataSpec.swift
+//  SwiftCheck
+//
+//  Created by Projector on 12/28/15.
+//  Copyright © 2015 Robert Widmann. All rights reserved.
+//
+
+import SwiftCheck
+
+extension NSData {
+    static var generator: Gen<NSData> {
+        return Gen<NSData>.pure(
+            String.arbitrary.generate.dataUsingEncoding(NSUTF8StringEncoding)!
```

Comment:
Oh, awesome! Thanks! Still wrapping my head around SwiftCheck and how to use it effectively in my code.